### DISCUSSION TITLE
How to add a comment to a page

### DISCUSSION THREAD

**Discussion by Jaeger87** [ramnes/notion-sdk-py#179] [Q&A]
Hi, I didn't found an example on how to add a comment to a notion page; I tried by my self something like this:

```python
def addTextCommentToNotion(pageID):
    new_comment = {
    "rich_text": [
      {
        "text": {
          "content": "Hello world"
        }
      }
    ]
}
    notion.comments.create(
        block_id=pageID,
        comment=new_comment
)
```

I got a bad request error with this code, i'm pretty sure that i provide a good pageID because the function to list comments works. What am I doing wrong?

**Comment by Jaeger87** [comment]
Solved by providing as parent an object with just the field "parent_id", can i suggest to make possible to pass directly a string with the parent id?

**Reply by ramnes** [reply]
I'm not sure to understand, can you provide a code example to show what you're suggesting?

**Reply by Jaeger87** [reply]
Right now to send a comment a do this:

`    return notion.comments.create(
        parent = {
    'page_id' : pageID},
        rich_text=new_comment
)
`

My suggestion is this:

`    return notion.comments.create(
        parent = pageID,
        rich_text=new_comment
)
`

**Reply by ramnes** [reply]
No, I'm sorry. The current goal of this library is to stay as close as possible to the official JS SDK and  API, and that's not how they work.
